packages/bpk-component-bottom-sheet/src/BpkBottomSheet.tsx

```diff
@@ -51,20 +50,22 @@ export type Props = {
   isOpen: boolean;
 }
 
-const BpkBottomSheet = ({
-  actionText = '',
-  ariaLabelledby,
-  children,
-  closeLabel = '',
-  closeOnEscPressed = false,
-  closeOnScrimClick = false,
-  id,
-  isOpen,
-  onAction = () => null,
-  onClose,
-  title = '',
-  wide = false
-}: Props) => {
+export type Props = CommonProps & ({ ariaLabelledby: string } | { ariaLabel: string; });
```

Comment:
Interesting, I thought interfaces could only be extended and `&` wouldn't work on them

Reply:
Yeah there is some subtlety there, you can mention interfaces when using intersection to define a `type` (you just can define an `interface` using intersection).
- https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces